src/CommandBlock.js

```diff
@@ -57,15 +55,32 @@ export default React.forwardRef<CommandBlockProps, Button>(
             ...otherProps
         } = props;
 
-        let icon = null;
+        let children = null;
         const iconType = commandBlockIconTypes.get(commandName);
         if (iconType) {
-            icon = React.createElement(
+            children = React.createElement(
                 iconType,
                 {
                     className: 'command-block-svg'
                 }
             );
+        } else {
```

Comment:
For our future selves, I think it would be better here to check explicitly that `commandName` is `startLoop` or `endLoop` to guard this rendering of the loop blocks, rather than assuming that we have a loop if the command isn't found in `commandBlockIconTypes`.